src/controller/controller.cpp

```diff
@@ -9,7 +9,7 @@ void Controller::init() {
     static Uniform distY = Uniform(0.0, world.HEIGHT);
     static Uniform distE = Uniform(MIN_FOOD_SIZE, MAX_FOOD_SIZE);
 
-    for (int i = 0; i < 10; i++) world.addIndividual(new Fred(10 * i, 10 * i, 0, 40));
+    world.addIndividual(new Piercy(0, 0, 0, 0.2 * world.ENERGY));
```

Comment:
you should add a switch for that instead of simply replacing the code

Reply:
resolved in #28